app/src/sharedTest/java/org/simple/clinic/util/TestFunctions.kt

```diff
@@ -29,6 +30,10 @@ fun randomGender(): Gender {
   return Gender.TypeAdapter.knownMappings.keys.shuffled().first()
 }
 
+fun randomDeletedReason(): DeletedReason {
+  return DeletedReason.TypeAdapter.knownMappings.keys.shuffled().first()
```

Comment:
I think Kotlin has an extension function now to get a random element from a collection. So `.shuffled().first()` could be replaced with `.random()`.